### DISCUSSION TITLE
WooCommerce Checkout Page

### DISCUSSION THREAD

**Discussion by batool-sajdi** [woocommerce/woocommerce#48200] [Checkout Flow]
do you lock is a block"

**Comment by ralucaStan** [comment]
Hi @batool-sajdi,
could you share more details and/or screenshots to support this discussion? It's unclear what your intention is. Thank you so much

**Comment by batool-sajdi** [comment]
*Hi *Raluca Stan
yes I understand share more details and/or screenshots to support this
discussion
Thank you.
